SPECS/kernel-hci/kernel-hci.spec

```diff
@@ -55,6 +55,7 @@ Patch24:        0025-net-mlx5-Bridge-implement-infrastructure-for-VLAN-pr.patch
 Patch25:        0026-net-mlx5-Bridge-implement-QinQ-support.patch
 Patch26:        0027-mstflint-This-driver-enables-under-the-secure-boot.patch
 Patch27:        0028-net-mlx5-Bridge-use-debug-not-warn-if-entry-not-found.patch
+Patch28:        0029-enable-dm-multipath-kernel-configs.patch
```

Comment:
This is for patching the kernel-hci source code. The config file is part of the source code of this repository, at SPECS/kernel-hci/config. So you can just make the desired changes there.